Index: kmm_ubi4/src/commonMain/kotlin/com/bailout/stickk/ubi4/data/state/BLEState.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.bailout.stickk.ubi4.data.state\n\nimport com.bailout.stickk.ubi4.data.parser.BLEParser\nimport kotlin.properties.Delegates\n\nobject BLEState {\n    var bleParser by Delegates.notNull<BLEParser>()\n\n}
===================================================================
diff --git a/kmm_ubi4/src/commonMain/kotlin/com/bailout/stickk/ubi4/data/state/BLEState.kt b/kmm_ubi4/src/commonMain/kotlin/com/bailout/stickk/ubi4/data/state/BLEState.kt
--- a/kmm_ubi4/src/commonMain/kotlin/com/bailout/stickk/ubi4/data/state/BLEState.kt	
+++ b/kmm_ubi4/src/commonMain/kotlin/com/bailout/stickk/ubi4/data/state/BLEState.kt	
@@ -1,9 +1,9 @@
-package com.bailout.stickk.ubi4.data.state
-
-import com.bailout.stickk.ubi4.data.parser.BLEParser
-import kotlin.properties.Delegates
-
-object BLEState {
-    var bleParser by Delegates.notNull<BLEParser>()
-
-}
\ No newline at end of file
+//package com.bailout.stickk.ubi4.data.state
+//
+//import com.bailout.stickk.ubi4.data.parser.BLEParser
+//import kotlin.properties.Delegates
+//
+//object BLEState {
+//    var bleParser by Delegates.notNull<BLEParser>()
+//
+//}
\ No newline at end of file
